src/code_80057C60.c

```diff
@@ -364,7 +366,6 @@ f32 D_8018CFCC;
 s16 D_8018CFD0;
 f32 D_8018CFD4;
 s16 D_8018CFD8;
-UNUSED s32 D_8018CFDC;
```

Comment:
This might be why it doesn't match now.

Reply:
it auto align in 0x8 so a file bss can only start with XXXXX8 or XXXXX0 so it's not needed here